.github/workflows/CI.yml

```diff
@@ -38,6 +38,16 @@ jobs:
         with:
           submodules: true
 
+      # Git functionality in CMAKE file does not work with given ubuntu image. Therefore, handling it here.
+      # Deleting file at the end to skip `git apply` inside CMAKE file
+      - name: Apply Git Patch
+        run: |
+          cd jni/external/faiss
+          git config --global core.autocrlf input
+          git apply --ignore-space-change --ignore-whitespace --3way ../../patches/faiss/0001-Custom-patch-to-support-multi-vector.patch
+          rm ../../patches/faiss/0001-Custom-patch-to-support-multi-vector.patch
```

Comment:
Why remove?

Reply:
Commented in line #42. `# Deleting file at the end to skip `git apply` inside CMAKE file`